lib/modules/platform/github/index.ts

```diff
@@ -1706,7 +1706,9 @@ export async function commitFiles(
   if (!pushResult) {
     return null;
   }
-  // Because the branch commit was done remotely via REST API, now we git fetch it locally.
-  // We also do this step when committing/pushing using local git tooling.
-  return git.fetchCommit(config);
+  // Replace locally created branch with the remotely created one
+  // and return the remote commit SHA
+  await git.resetToCommit(commitResult.parentCommitSha);
```

Comment:
does it do a git fetch before reset? otherwise it's a no-op

Reply:
Earlier in this function we've created local commit that has been used for the sake of obtaining trees and blobs required to construct remote commit via GitHub API. Once it's created, we have to pull it, and this `resetToCommit` just removes the initial commit, so that we don't have any conflicts.

Reply:
ok, just wanted bro verify that we don't silently miss the pull. 🙃